rua.env

```diff
@@ -0,0 +1,7 @@
+DATABASE_HOST=mysql
+DATABASE_NAME=rua
+DATABASE_USER=rua
+DATABASE_PASS=pass
+DJANGO_SETTINGS_MODULE=core.settings_dev
+PYTHONPATH=/rua/src
+TEST_ENVIRONMENT=True
```

Comment:
We should also copy/paste these very variables in the main Dockerfile, so we have a documentation of what ENV variables are used in the Docker container.